src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -62,6 +65,43 @@ class ServiceTypeAhead extends Component {
     return false;
   };
 
+  renderSearchForAvailableServicePrompt = inputValue => {
+    const { isFocused } = this.state;
+    const { showError } = this.props;
+
+    if (
+      (isFocused && !inputValue && !showError) ||
+      (inputValue && inputValue.length < 2)
+    ) {
+      return (
+        <div className="dropdown" role="alert">
+          <div className="vads-u-margin--1p5">
+            {'Search for an available service'}
+          </div>
+        </div>
+      );
+    } else return null;
+  };
+
+  renderTryAnotherServicePrompt = inputValue => {
+    const { services } = this.state;
+
+    if (
+      inputValue &&
+      inputValue.length >= 2 &&
+      !services.filter(specialty => this.shouldShow(inputValue, specialty))
```

Comment:
We might want to refactor this into a function:

```js
matchingServices = inputValue =>
      this.state.services.filter(specialty => this.shouldShow(inputValue, specialty))
```

.. and use it on line 169 as well.

Reply:
I can do this